docs/components/connectors/use-connectors/inbound.md

```diff
@@ -48,8 +48,8 @@ You can still start instances of that process manually via the modeler, which is
 
 ### Modeling the Connector non-interrupting message start event
 
-1. Start building your BPMN diagram with an **Event Sub Process**.
-2. Add a plain **Message Start Event (non-interrupting)** into an **Event Sub Process**.
+1. Start building your BPMN diagram with an **Event subprocess**.
```

Comment:
<img width="256" alt="Screenshot 2024-03-11 at 1 22 04 PM" src="https://github.com/camunda/camunda-docs/assets/84338309/63717987-f6ac-490f-ac26-708f8c015e09">
Following up with Modeler on this 👍